src/server.cpp

```diff
@@ -111,10 +118,6 @@ namespace quicr {
 
                 // TODO(tievens): add filter type when caching supports it
                 SubscribeReceived(conn_ctx.connection_handle, msg.subscribe_id, msg.track_alias, tfn, {});
-
-                // TODO(tievens): Delay the subscribe OK till ResolveSubscribe() is called
-                SendSubscribeOk(conn_ctx, msg.subscribe_id, kSubscribeExpires, false);
```

Comment:
Removing this would require it to be added to resolve subscribe, right?  I don't see that changed added.

Reply:
good catch . That was error . I will add it back